aries-site/src/pages/templates/lists.mdx

```diff
@@ -1,16 +1,109 @@
 import { Example } from '../../layouts';
-import { ListScreenExample } from '../../examples';
+import {
+  ListIconIdentifierExample,
+  ListImageIdentifierExample,
+  ListNameOptionActionExample,
+  ListNameActionExample,
+  ListNameDescriptionOptionExample,
+  ListScreenExample,
+} from '../../examples';
 
 HPE Design System list view templates are go-to patterns for displaying homogeneous data such as services, devices, users, and more.
-
-List views are optimized for scanability and reading comprehension. Each list item provides users focussed information and identity labels to aid selection, decision making, and action.
-
-## Manage Users Screen
+List views are optimized for scanability and reading comprehension. Each list item provides users focused information and identity labels to aid selection, decision making, and action.
 
 <Example
   code="https://raw.githubusercontent.com/hpe-design/design-system/master/aries-site/src/examples/templates/list-views/ListScreenExample.js"
+  docs="https://v2.grommet.io/list?theme=hpe#props"
   template
   width="100%"
 >
   <ListScreenExample />
 </Example>
+
+## Guidance
+
+Lists are an effective way to display dense content to users. Here are some recommendations on how to make the data easily consumable.
+
+**Best Practices:**
+
+- Be consistent with layout and visual treatments for all list items.
+- Keep list item descriptions brief and concise.
+- Use Icons or Images in list items to help draw attention and support scanning.
+- Maintain sufficient space between list items or consider using dividers between rows.
+- Sort lists in a logical order such as alphabetically, numerically, chronologically, or by user preference.
+
+**Lists Types:**
+
+- Single-line lists are optimal for scan-ability, especially with the addition of an image.
+- Two-line lists provide a primary and secondary text which can be helpful when more detail is needed.
+- Three-line lists of a primary text and two lines of secondary text are not recommended.
+- Grid lists are made of image tiles and require more space.
+
+## Variants
+
+Below are some common list variations that demonstrate how to combine visual content, text, and list actions within the component.
+
+### Image + Name + Description + Action
+
+Two-line list with an image, primary title, description, and an action button.
+
+<Example
```

Comment:
Do we provide any guidance as to a max-width for a List? On widescreens it becomes difficult to track from left to righthand side:
![Screen Shot 2020-07-06 at 1 38 59 PM](https://user-images.githubusercontent.com/1756948/86633643-27126680-bf8e-11ea-94b1-da89b6ca32b0.png)

Reply:
At one point, I thought we had list examples with a bottom border. Did that get eliminated?

Reply:
That's a great point. We should have some guidance on this.

Reply:
> Also, regarding bottom border, the Figma designs now call for now border. However, I will reach out to see if we want to demonstrate any cases with a border and look into implementation. Since the border removal will be handled in the theme, it seems like we'd want border to be accessible via props to avoid use of Theme.Extend

Clarifying:
- We will handle the most common execution of border (no border) in the theme.
- If a border is desired, we will use [List's border prop](https://v2.grommet.io/list#border)

Reply:
We will want to add guidance on when List with border would be appropriate as well. I'm not sure on the guidance for when to use border or not, so I will reach out to the design team.

Reply:
Need to change the primary example on this page. It uses outdated styling. Pick one of the other examples to show instead.

BTW - I filed #974 - Update Figma Designs for Lists tesmplate - to get some help from either Greg or Vicky on updates to the Figma designs for this.